app/javascript/packs/new_timelog.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import Select from 'react-normalized-select'
+
+class NewTimelog extends React.Component {
+
+  constructor(props) {
```

Comment:
Pass `...args` to super from the constructor. Look above for reference why and how to do this.